### PR TITLE
Sign in sign up page

### PR DESCRIPTION
**Pull request by omkarpatke**
added signIn, signUp pages

### CODE REVIEW

File: components/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.product{
+    display: flex;
+    line-height: 1.7;
+    padding: 1rem;
+    margin: 1rem 1rem;
+    width: auto;
+    height: auto;
+    border-top: 1px solid #DDD;
```
Comment (sri-rishi) [inline components/cart/cart.css:33]: use variable color naming

File: components/cycles/cycles.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+/* products */
+
+.products-container{
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    gap: 1.5rem 1.5rem;
+    padding: 0rem 1rem;
```
Comment (sri-rishi) [inline components/cycles/cycles.css:39]: can be used 0 instead of  0rem

File: components/signUp/signUp.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    font-size: 1.8rem;
+}
+
+.login-card{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
+    padding: 1rem 3rem 2rem 3rem;
+    margin: 1rem 0 3rem 0;
```
Comment (sri-rishi) [inline components/signUp/signUp.css]: can use shorthand property of margin: 1rem 0 3rem

File: components/signUp/signUp.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    color: var(--primary-color);
+    margin-bottom: 1.4rem;
+}
+
+.forgot-password-link:hover{
+    color: var(--danger-color);
+}
+
+.login-btn{
+    padding: 8px 2.4rem 8px 2.4rem;
```
Comment (sri-rishi) [inline components/signUp/signUp.css]: can use shorthand property of padding: 8px 2.4rem

File: components/signUp/signUp.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    transition: .8s all ease-in-out;
+    cursor: pointer;
+}
+
+.login-btn:hover{
+    background-color: var(--black-color);
+}
+
+.signUp-with-google-btn{
+    padding: 5px 2.4rem 5px 2.4rem;
```
Comment (sri-rishi) [inline components/signUp/signUp.css]: here also  can used shorthand

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Just a suggestion can use label with inputs, good for accessibility

File: components/signUp/signUp.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+.login-container h3{
+    padding-top: 1rem;
+    font-size: 1.8rem;
+}
+
+.login-card{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
```
Comment (rahulk31) [inline components/signUp/signUp.css:21]: You can plan to use DRY principle. Here in login-container and login-card classes the flex properties are repeating. So you can make a separate class for that altogether.

**Review by rahulk31 — APPROVED** [review]
Overall code is good and class names are well used. Good to merge.

**Pull request merged**